app/models/bank_account.rb

```diff
@@ -47,7 +47,7 @@ def set_initial_residue
     transactions.create(amount_cents: residue_cents, transaction_type: 'Residue') if residue > 0
   end
 
-  def to_s
-    "#{name} (#{Money::Currency.new(currency).symbol})"
+  def to_label
```

Comment:
@Gataniel why did you rename method? Leave it as is.

Instead truncating whole string, truncate only name, so currency symbol will be still visible.